phpBB/phpbb/passwords/driver/base_native.php

```diff
@@ -57,6 +57,25 @@ public function hash($password)
 		return password_hash($password, $this->get_algo_value(), $this->get_options());
 	}
 
+	/**
+	* Check if Sodium implementation for argon2 algorithm is being used
+	*
+	* @link https://wiki.php.net/rfc/sodium.argon.hash
+	*
+	* @return bool
+	*/
+	public function is_sodium()
+	{
+		static $is_sodium;
+
+		if (empty($is_sodium))
```

Comment:
I would use isset better instead of empty for undefined variables

Reply:
Makes perfect sense for bool var. Good catch.